src/category_theory/closed/monoidal.lean

```diff
@@ -108,7 +108,7 @@ lemma coev_naturality {X Y : C} (f : X ⟶ Y) :
   f ≫ (coev A).app Y = (coev A).app X ≫ (ihom A).map ((𝟙 A) ⊗ f) :=
 (coev A).naturality f
 
-notation A ` ⟶[C] `:20 B:19 := (ihom A).obj B
+notation A ` ⟶[`C`] ` B:10 := (@ihom C _ _ A _).obj B
```

Comment:
This is part of another PR, right?

Reply:
Yes, it's from the dependent PR #12612 which has since been merged.